src/protocolgame.cpp

```diff
@@ -902,6 +903,14 @@ void ProtocolGame::parseFollow(NetworkMessage& msg)
 	addGameTask(&Game::playerFollowCreature, player->getID(), creatureId);
 }
 
+void ProtocolGame::parseEquipObject(NetworkMessage& msg)
+{
+	uint16_t spriteId = msg.get<uint16_t>();
+	// msg.get<uint8_t>();
```

Comment:
Mark asked in the previous PR to just comment it and let it sit there so we know there's something being sent